src/Interpreters/Cache/FileCache.cpp

```diff
@@ -889,6 +896,9 @@ bool FileCache::tryReserve(
     if (!main_priority->collectCandidatesForEviction(
             size, required_elements_num, reserve_stat, eviction_candidates, queue_iterator, user.user_id, cache_lock))
     {
+        LOG_DEBUG(log, "Collection candidates for eviction from cache is unsuccessful, "
```

Comment:
Having any logging here would be too verbose while this log itself does not represent much benefit for debugging.